src/handlers/comment/commands.ts

```diff
@@ -1,7 +1,7 @@
 export enum IssueCommentCommands {
   HELP = "/help", // list available commands
-  ASSIGN = "/assign", // assign the hunter to the issue automatically
-  UNASSIGN = "/unassign", // unassign to default
+  ASSIGN = "/start", // assign the hunter to the issue automatically
+  UNASSIGN = "/stop", // unassign to default
```

Comment:
still seeing `ASSIGN` and `UNASSIGN`. They also need to be replaced

Reply:
oh sorry about that!

Reply:
did you push something @me505 ? I don't see any commits after the conversation.